j5/components/piezo.py

```diff
@@ -0,0 +1,60 @@
+"""Classes for Piezo support."""
+
+from abc import ABCMeta, abstractmethod
+from datetime import timedelta
+from enum import Enum
+from typing import TypeVar
+
+from j5.boards import Board
+from j5.components import Component
+
+
+class Note(Enum):
+    """An enumerator of frequencies in Hz."""
+
+    C6 = 1046.50
+    D6 = 1174.66
+    E6 = 1318.51
+    F6 = 1396.91
+    G6 = 1567.98
+    A6 = 1760.00
+    B6 = 1975.53
+    C7 = 2093.00
+    D7 = 2349.32
+    E7 = 2637.02
+    F7 = 2793.83
+    G7 = 3135.96
+    A7 = 3520.00
+    B7 = 3951.07
+
+
+Pitch = TypeVar('Pitch', int, Note)
```

Comment:
```suggestion
Pitch = Union[int, Note]
```

Reply:
I couldn't find the use case for 'TypeVar' vs 'Union'. What are the differences?

Reply:
`TypeVar`s are generally used with [Generics](https://mypy.readthedocs.io/en/latest/generics.html#defining-generic-classes) whereas `Union`s are used to state that it can be one of the specified types.

Reply:
Thank you 😊